### PR TITLE
Update ttyd and libwebsockets deps for ssh addon

### LINKED ISSUE #2186: Terminal&SSH addon unreadble CLI in Safari
## The problem

Text can't be read at all in the Terminal addon in Safari.

## Environment
Terminal&SSH addon v9.1.3, HA 2021.9.6, Safari on MacOS Monterey beta, also happens in Safari on Big Sur 11.6.

- Add-on with the issue: Terminal&SSH
- Add-on release with the issue: 9.1.3
- Last working add-on release (if known): 
- Operating environment (OS/Supervised): HAOS 6.3 on Rpi4

## Problem-relevant configuration

```yaml

```

## Traceback/Error logs

```txt

```

## Additional information

This only happens in Safari, Edge on the same machine is perfectly fine
Screenshot 2021-09-19 at 10 02 43
.

### PR DESCRIPTION
**Pull request by bval**
In Safari 15 and later, the web terminal for the ssh add-on no longer renders properly. This bumps the libwebsockets version and ttyd version to correct that. ttyd has not had a release since January of this year, so this pins the update to the same SHA in use for the community add-on, where this fix is ported from and known to work.

[Original issue report from community](https://github.com/hassio-addons/addon-ssh/issues/320)
[Original PR resolving above issue](https://github.com/hassio-addons/addon-ssh/pull/340)
Closes #2186

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @bval, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;addons#2324).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by pvizeli** [comment]
Changelog + bump version 0.1.0

File: ssh/build.yaml [hunk 1]
```diff
@@ -9,5 +9,5 @@ codenotary:
   base_image: notary@home-assistant.io
 args:
   CLI_VERSION: 4.14.0
-  LIBWEBSOCKETS_VERSION: 4.1.4
-  TTYD_VERSION: 1.6.3
+  LIBWEBSOCKETS_VERSION: 4.2.1
+  TTYD_VERSION: 62a5b635c6718109c64a6d629014f2784959f02c
```
Comment (ludeeus) [inline ssh/build.yaml:13]: What's the reason for this old commit?
Later commits in the repo fixes issues with webgl2

Comment (bval) [reply ssh/build.yaml:13]: @ludeeus Happy to go with a later commit, it looks like the community version of the ssh addon has moved forward to `3e37e33b1cd927ae8f25cfbcf0da268723b6d230`. I'll move this PR up as well.

**Comment by bval** [comment]
> Changelog + bump version 0.1.0

@pvizeli 👍

**Review by frenck — APPROVED** [review]
I've been running this combination in the community add-on for a while now; most issues around this are gone now.

**Review by ludeeus — APPROVED** [review]

**Pull request merged**